packer/install-dependencies.sh

```diff
@@ -357,6 +358,7 @@ K8S_VERSIONS="
 1.12.8
 1.12.7
 1.12.7-azs
+1.11.10
```

Comment:
@jadarsie @honcao FYI you might want to add the azure stack equivalent

Reply:
@CecileRobertMichon Thanks for the notification.
@rjaini FYI, we would need image for 1.11.10

Reply:
any updates on getting 1.11.10 out for Azure Stack?

Reply:
Since this payload has new API version changes related to Azure cloud provider. We need to validate more thoroughly. We will update once done. Mostly by Monday we should be done.
@jadarsie told me that this is not blocking. Is that blocking ?.

Reply:
not blocking, just trying to see if we should wait for it or go ahead and merge this PR without it and then do a follow up for Azure Stack.

Reply:
Let's not block. Ostensibly the extra docker image overhead is a no-op in terms of cleanup (i.e., all unneeded, extra images will be reliably cleaned up at provision time). Also, we have a storage alert in place, so we'll know if we run out of local storage to accommodate extra -azs images.